projects/1.16/assets/dungeons-gear/dungeons_gear/lang/zh_cn.json

```
@@ -775,5 +843,12 @@
   "ability.dungeons_gear.thrust_attack": "突袭",
   "ability.dungeons_gear.thundering": "有概率发射雷电（引雷 I）",
   "ability.dungeons_gear.weakening": "削弱敌人的攻击（虚弱 I）",
-  "ability.dungeons_gear.wind_up_attack": "发条射击"
-}
\ No newline at end of file
+  "ability.dungeons_gear.wind_up_attack": "发条射击",
+  "item.dungeons_gear.arrow_bundle": "箭束",
+  "ability.dungeons_gear.arrow_hoarder": "每捆+1支箭（弓箭囤积者I）",
+  "dungeons_gear.gilded": "镀金的",
+  "key.dungeons_gear.curiosintegration.category": "地下城装备饰品",
+  "key.dungeons_gear.curiosintegration.description_slot1": "饰品栏1",
+  "key.dungeons_gear.curiosintegration.description_slot2": "饰品栏2",
+  "key.dungeons_gear.curiosintegration.description_slot3": "饰品栏3"

```

```suggestion
  "key.dungeons_gear.curiosintegration.description_slot1": "法器栏1",
  "key.dungeons_gear.curiosintegration.description_slot2": "法器栏2",
  "key.dungeons_gear.curiosintegration.description_slot3": "法器栏3"

```
应该指的是与Curios模组的联动，翻译成法器栏似乎好一些。

---

采纳